### PR TITLE
ROX-15587: scrape infra-server metrics with a service monitor

### PR DESCRIPTION
**Pull request by tommartensen**
Proof from PR cluster:

Screenshot 2024-03-15 at 13 23 11

### CODE REVIEW

File: chart/infra-server/templates/monitoring/servicemonitors.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: infra-server
+  namespace: infra
+spec:
+  endpoints:
+  - port: metrics
+    scheme: https
```
Comment (BradLugo) [inline chart/infra-server/templates/monitoring/servicemonitors.yaml:22]: Should this be `http`?

Comment (tommartensen) [reply chart/infra-server/templates/monitoring/servicemonitors.yaml:22]: No: https://github.com/stackrox/infra/blob/master/server/server.go#L99-L105 .

Comment (tommartensen) [reply chart/infra-server/templates/monitoring/servicemonitors.yaml:22]: It's a TLS endpoint with a self-signed certificate.

Comment (BradLugo) [reply chart/infra-server/templates/monitoring/servicemonitors.yaml:22]: Ah, makes sense. I'll unblock the PR, but would it be worth configuring the CA in the `tlsConfig`? I can't remember if we have that available on the cluster

**Review by BradLugo — APPROVED** [review]
One question regarding TLS. Otherwise, LGTM. Thanks Tom! I'm excited to get more metrics and monitoring for infra 🙂

File: chart/infra-server/templates/monitoring/servicemonitors.yaml (hunk 1 above)
Comment (tommartensen) [reply chart/infra-server/templates/monitoring/servicemonitors.yaml:22]: Probably, but may be a bigger story.

**Pull request merged**